framework/widget/src/main/java/org/apache/ofbiz/widget/renderer/macro/MacroFormRenderer.java

```diff
@@ -3005,6 +3006,9 @@ public void makeHyperlinkString(Appendable writer, String linkStyle, String targ
             if (UtilValidate.isNotEmpty(request.getAttribute("id"))) {
                 id = request.getAttribute("id").toString();
             }
+            if (UtilValidate.isNotEmpty(request.getAttribute("title"))) {
+                imgTitle = request.getAttribute("title").toString();
```

Comment:
We should also take this opportunity to rename the variable from `imgTitle` to `title`, better indicating the intent of the variable to the reader.